examples/handlebars_templates/src/main.rs

```diff
@@ -39,10 +40,22 @@ fn not_found(req: &Request) -> Template {
     Template::render("error/404", &map)
 }
 
+struct EchoHelper;
```

Comment:
You shouldn't need a struct here. You should be able to pass a function with the following signature to the `register_helper` method directly

```rust
(&Helper, &Registry, &mut RenderContext) -> HelperResult;
```